addons/website_sale/models/product_template.py

```diff
@@ -190,14 +191,32 @@ def _get_sales_prices(self, pricelist):
                 'price_reduce': price_reduce
             }
 
-            if show_discount:
+            has_compare_list_price = template.compare_list_price
+            has_pricelist_price = pricelist_contains_templates[template.id]
```

Comment:
Same here no need to store it in a variable when you can access it directly from the `dict`